app/controllers/authenticate_controller.rb

```diff
@@ -128,23 +128,24 @@ def k8s_inject_client_cert
   private
 
   def handle_login_error(err)
-    logger.debug("Login Error: #{err.inspect}")
+    logger.info("Login Error: #{err.inspect}")
```

Comment:
I think it is confusing to have a `login error` in INFO level. maybe it will be better to write `login failed` as this is the information you want to give.
Unless it is really an error and not a valid part of the business logic

Reply:
it is not an error but customers (and SEs) already know to look for these messages so i don't want to change that.